cmd/image-builder/github-workflow-integration.md

```diff
@@ -0,0 +1,156 @@
+### Workflow Identification Claims
+
+The OIDC token contains the following claims that can be used to identify the workflow that triggered the build pipeline. These include:
+
+- `iss`: The issuer of the token. This is always `https://token.actions.githubusercontent.com`.
+- `iat`: The time at which the token was issued.
+- `exp`: The time at which the token expires.
+- `jti`: A unique identifier for the token.
+- `nbf`: The time before which the token must not be accepted for processing.
+- `kid`: The key ID of the key used to sign the token.
+- `alg`: The algorithm used to sign the token.
+- `run_id`: The ID of the workflow run.
+- `run_number`: The number of the workflow run.
+- `actor`: The login of the user who initiated the workflow run.
+- `event_name`: The name of the event that triggered the workflow run.
+- `workflow`: The name of the workflow that triggered the workflow run.
+- `workflow_ref`: The git ref associated with the workflow file.
+- `repository`: The repository where the workflow run occurred.
+- `repository_owner`: The owner of the repository where the workflow run occurred.
+
+### Source Code Cloning Claims
+
+The OIDC token also contains claims that can be used to clone the appropriate version of the source code:
+
+- `repository`: The repository where the workflow run occurred.
+- `repository_owner`: The owner of the repository where the workflow run occurred.
+- `event_name`: The name of the event that triggered the workflow run.
+- `ref`: The git ref associated with the workflow run. This can be used to checkout the correct branch, tag, or commit.
+- `ref_type`: The type of git ref associated with the workflow run. This can be used to determine whether the ref is a branch, tag, or
+  commit.
+- `base_ref`: The base git ref associated with the workflow run. This can be used to determine the base branch for a pull request.
+- `head_ref`: The head git ref associated with the workflow run. This can be used to determine the head branch for a pull request.
+
+These claims ensure that the `oci-image-builder` pipeline builds the exact version of the code that was provided in the PR or merged to the
+branch, adhering to slc-29 compliance.
+
+## Parameters for the `oci-image-builder` Pipeline
+
+The `oci-image-builder` pipeline requires certain data to be provided in parameters to execute a build process.
+Certain parameters need to be defined by the user additionally to the data taken from OIDC token.
+
+These parameters include:
+
+### User Defined Parameters
+
+- `Context`: The context of the build.
+- `Dockerfile`: The Dockerfile to be used for the build.
+- `Name`: The name of the image.
+- `BuildArgs`: The build arguments to be passed to the build.
+- `Tags`: The tags to be applied to the image.
+- `ExportTags`: Whether to export the tags.
+
+### Parameters from OIDC Token
+
+- `RepoName`: The name of the repository.
+- `RepoOwner`: The owner of the repository. Possible values include "kyma-project" and "kyma-incubator".
```

Comment:
kyma-incubator will be owned by compass team and I am not sure if they should be included into new image builder as they should migrate out of Prow until end of June. I think this is question for @TorstenD-SAP .

Reply:
When `kyma-incubator` org will not be allowed to build images with our solution, we will simply remove respective value from allowed org list.